README.md

```diff
@@ -141,6 +142,35 @@ render(<App />, document.getElementById("root"));
 
 Here's the full version: [https://codesandbox.io/s/n5orzr5mxj](https://codesandbox.io/s/n5orzr5mxj)
 
+## Middleware
+
+The middleware has a method signature that is almost similar to a traditional redux middleware.
+
+```js
+/* logger.js */
+const logger = (store) => (next) => (action, name) => {
+  console.log('action called', name)
+  return next(action);
+  // or can totally change actions as well.
+  // eg: return next(doSomeOtherAction);
+}
+```
+
+```js
+/* store.js */
+import createStore from "redux-zero";
+import { applyMiddleware } from "redux-zero/utils"
```

Comment:
Why the `utils` folder? What you think about

```javascript
import { applyMiddleware } from "redux-zero/middleware"
```

?

Reply:
mentally, I treated all these as a utility of redux-zero. but middleware makes sense as well. I've moved it.